spec/cypress/e2e/simple_spec.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('My First Test', function() {
```

Comment:
Is that related to the PR?

Reply:
No, this is not at all related to this PR. By doing a

```
git log -p -- ./spec/cypress/e2e/simple_spec.cy.js
```

from the current branch, I found out that the file was introduced when I merged `mampf-next` into this branch earlier on, see [here](https://github.com/MaMpf-HD/mampf/commit/843854fe8317103b12b78e903a10ae17d59de77c#diff-f971a7862ba60ddb68800528aa4209773ddb3622a1507a123b0afc5aa3271a98). The file was renamed, so it popped up here as well. However, meanwhile I cannot find the file in the `mampf-next` branch anymore which is a bit strange as `git merge mampf-next` just gives me an "Already up to date".

Found out that the file was deleted on mampf-next [in this commit](https://github.com/MaMpf-HD/mampf/commit/a0557b868b397d85d72fea4d5b73566c192393a7#diff-f971a7862ba60ddb68800528aa4209773ddb3622a1507a123b0afc5aa3271a98) by running this command (from the mampf-next branch):
```
git rev-list -n 1 HEAD -- ./spec/cypress/e2e/simple_spec.cy.js
```

This is all a bit weird. To replay my merge and find out why the file was added there I did a:
```
git checkout 843854^
git merge mampf-next
```

I faced merge conflicts again but the `simple_spec.cy.js` file was not touched. So in short, I don't know why I readded this file in my merge commit. I will just delete it as not needed.